client/consensus/aura/src/lib.rs

```diff
@@ -788,13 +788,14 @@ impl<Block: BlockT, C, I, P> BlockImport<Block> for AuraBlockImport<Block, C, I,
 }
 
 /// Start an import queue for the Aura consensus algorithm.
-pub fn import_queue<B, I, C, P>(
+pub fn import_queue<B, I, C, P, F>(
 	slot_duration: SlotDuration,
 	block_import: I,
 	justification_import: Option<BoxJustificationImport<B>>,
 	finality_proof_import: Option<BoxFinalityProofImport<B>>,
 	client: Arc<C>,
 	inherent_data_providers: InherentDataProviders,
+	spawner: F,
```

Comment:
there's a case for using `impl Trait` here instead of adding an explicit generic param

Reply:
Yup this was my first attempt however rustc complained:
```
cannot provide explicit generic arguments when `impl Trait` is used in argument position
```
[for this call](https://github.com/paritytech/substrate/blob/4b911072bfe36652c8d3874cb8020f7ff426abed/bin/node-template/node/src/service.rs#L210).
Let me know if there's a cleaner way out! It was my first time trying out `impl Trait` so maybe I'm missing something :p

Reply:
That seems like a legitimate concern, so it's fine as-is!